apps/prairielearn/src/migrations/20230531184500_assessment_access_policies.sql

```diff
@@ -0,0 +1,19 @@
+-- Create assessment_access_policies table
+CREATE TABLE IF NOT EXISTS
+  assessment_access_policies (
+    id BIGSERIAL PRIMARY KEY,
+    assessment_id BIGINT REFERENCES assessments (id) ON UPDATE CASCADE ON DELETE CASCADE NULLS NOT DISTINCT,
```

Comment:
I believe we want this - should there ever be an `assessment_access_policies` row that isn't associated with an assessment?

```suggestion
    assessment_id BIGINT NOT NULL REFERENCES assessments (id) ON UPDATE CASCADE ON DELETE CASCADE,
```

Don't forget to update the schema in `database/tables` after making this change.